<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1221</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>911</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="Button_Amont_Aval_3">
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>--&gt; Amont vers Aval --&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Aval_Amont_3">
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;-- Aval vers Amont &lt;--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/Schéma.gif</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Mode: Automatique</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/stop.png</normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>540</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Authentification</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/lock.png</normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/lock.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Mode: Automatique</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>200</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="value">
       <number>33</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/stop.png</normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="value">
       <number>66</number>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>911</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Lorsque vous êtes dans le sas:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Amont_Aval_4">
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Fermer la porte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/Schéma.gif</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>540</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Authentification</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/lock.png</normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/lock.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-11</y>
        <width>1231</width>
        <height>611</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>260</y>
         <width>401</width>
         <height>340</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="Button_1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="Button_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="Button_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Button_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Button_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Button_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Button_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Button_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="Button_ok">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Button_0">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Button_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="Button_rm">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;--</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="Champ_mdp">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>180</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Ubuntu&quot;;
text-align: center;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>901</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Ubuntu&quot;;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Saisissez votre mot de passe pour vous connecter:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_retour">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>30</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string> Retour</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/retour.png</normaloff>../build-tp_note-Desktop_Qt_5_8_0_GCC_64bit-Debug/ressources/retour.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
